service/switchboardapp/src/test/java/com/switchboard/app/fixture/DeviceFixture.java

```diff
@@ -0,0 +1,24 @@
+package com.switchboard.app.fixture;
+
+import com.switchboard.app.domain.DeviceEntity;
```

Comment:
`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated